nfcsedev.x

```diff
@@ -0,0 +1,201 @@
+/*++
```

Comment:
You probably want to apply these changes to this file https://github.com/Microsoft/NFC-Class-Extension-Driver/blob/master/inc/NfcSEDev.h

Reply:
I think I misunderstood. I will change that.